ifrs17/Report/ReportScopes.ipynb

```diff
@@ -275,12 +275,12 @@
                 "\n",
                 "\npublic interface LockedBestEstimate: IScope<(ReportIdentity Id, CurrencyType CurrencyType), ReportStorage>, IDataCube<ReportVariable> {",
                 "\n    IDataCube<ReportVariable> LockedBestEstimate => GetScope<FxData>((Identity.Id, Identity.CurrencyType, EstimateTypes.BE)).FxData",
-                "\n        .Filter((\"LiabilityType\", Identity.Id.LiabilityType), (\"EconomicBasis\", EconomicBases.L));",
+                "\n        .Filter((\"LiabilityType\", Identity.Id.LiabilityType), (\"EconomicBasis\", EconomicBases.L), (\"AmountType\", \"!CDRI\"));",
```

Comment:
Dear @tkukuljan, it is possible that we need to carry this change in the client project.

Reply:
In general, treatment of CDR should include CDRI, let's talk about it.